add.go

```diff
@@ -88,6 +88,11 @@ func getURL(src string, chown *idtools.IDPair, mountpoint, renameTarget string,
 		return err
 	}
 	defer response.Body.Close()
+
+	if response.StatusCode < 200 || response.StatusCode >= 400 {
+		return errors.Errorf("invalid response status %d", response.StatusCode)
```

Comment:
```suggestion
		return fmt.Errorf("invalid response status %d", response.StatusCode)
```
Could we add the URL to the error message as well?